### PR TITLE
Various updates to TT1 Blocks

### PR DESCRIPTION
**Pull request by carolinan**
See comments

### CODE REVIEW

File: tt1-blocks/README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # TT1 Blocks
 
-Ongoing development for the block-based version of Twenty Twenty-One, the default WordPress theme slated for 5.6. 
+Ongoing development for the block theme version of Twenty Twenty-One.
 
-This theme requires the Gutenberg Plugin to be installed (preferably version 9.3 or later).
+This theme requires the Gutenberg Plugin to be installed (Version 10.6 or later).
 
-For the non-block-based version of this theme, please [visit the Twenty Twenty-One repository](https://github.com/WordPress/twentytwentyone).
```
Comment (carolinan) [inline tt1-blocks/README.md:7]: The linked repository is archived.

File: tt1-blocks/inc/block-styles.php [hunk 2]
```diff
@@ -36,15 +35,6 @@ function twenty_twenty_one_register_block_styles() {
...
 
-		// Group: Borders.
-		register_block_style(
-			'core/group',
-			array(
-				'name'  => 'twentytwentyone-border',
-				'label' => esc_html__( 'Borders', 'tt1-blocks' ),
-			)
-		);
-
```
Comment (carolinan) [inline tt1-blocks/inc/block-styles.php:47]: This style is replaced with the border support for the group block.

File: tt1-blocks/theme.json [hunk 3]
```diff
@@ -175,7 +176,20 @@
...
 		},
 		"spacing": {
-			"customPadding": true
+			"customPadding": true,
+			"units": [
+				"px",
+				"em",
+				"rem",
+				"vh",
+				"vw"
```
Comment (carolinan) [inline tt1-blocks/theme.json:185]: This replaces the theme support that was removed from functions.php

File: tt1-blocks/theme.json [hunk 4]
```diff
@@ -282,13 +296,7 @@
 					"lineHeight": "var(--wp--custom--line-height--body)"
 				}
 			},
-			"core/post-hierarchical-terms": {
-				"typography": {
-					"fontSize": "var(--wp--preset--font-size--extra-small)",
-					"lineHeight": "var(--wp--custom--line-height--body)"
-				}
-			},
-			"core/post-tags": {
```
Comment (carolinan) [inline tt1-blocks/theme.json:291]: These two blocks were reduced to one and renamed.

File: tt1-blocks/block-templates/index.html [hunk 5]
```diff
@@ -31,9 +31,9 @@
 		<!-- /wp:column -->
 
 		<!-- wp:column -->
-		<div class="wp-block-column"><!-- wp:post-hierarchical-terms {"term":"category","textAlign":"right"} /-->
+		<div class="wp-block-column"><!-- wp:post-terms {"term":"category","textAlign":"right"} /-->
 
-		<!-- wp:post-tags {"textAlign":"right"} /--></div>
+		<!-- wp:post-terms {"term":"post_tag","textAlign":"right"} /--></div>
 		<!-- /wp:column --></div>
```
Comment (carolinan) [inline tt1-blocks/block-templates/index.html:37]: These two blocks were reduced to one and renamed.

File: tt1-blocks/theme.json [hunk 6]
```diff
@@ -175,7 +176,20 @@
 			]
 		},
 		"spacing": {
-			"customPadding": true
+			"customPadding": true,
+			"units": [
+				"px",
+				"em",
+				"rem",
+				"vh",
```
Comment (jffng) [inline tt1-blocks/theme.json:184]: For some reason `vh` does not show up for me:

Screen Shot 2021-05-10 at 8 42 58 PM

**Review by jffng — APPROVED** [review]
LGTM, just one small comment in my testing.

**Pull request merged**